public/graphql/components/get_all.graphql

```diff
@@ -1,24 +1,27 @@
-query Components {
+query {
   atoms: admin_liquid_partials(
-    filter: {path: {contains: "components/atoms/"}}
+    filter: { path: { contains: "components/atoms/" } }
```

Comment:
I've started to wonder, maybe instead of trying to identify atoms/molecules/etc. via a path, we could add to `metadata` something like `component_type: atom` ? From the performance perspective, `contains` translates to `%<val>%` in SQL which as we know can't use the DB index and can become a bottleneck. Since we might need similar queries in other modules, it would be good to make it as efficient as possible. I've considered also using `starts_with` which would translate to `val%` which would be much better, however, nothing beats an exact match + starts_with will not work with overwrites. Looking forward to your thoughts!

Reply:
You are right @Slashek. I'm just thinking about that this query is only used by the styleguide page, so I'm not sure that we need to modify the metadatas because component library will be used with `theme_render` tag, so I think it won't cause any performance issues. What do you think?

Reply:
So this will become an issue for the /styleguide page if our channel partners would create a site with more than 100k+ partials, internally we shouldn't have this problem (arabbank for example uses 1.5k partials). Does not seem like a critical improvement to do right away in this scenario, please only note we'd like to avoid `contains` as much as possible in general

Reply:
Thanks for the note, @Slashek!